### ISSUE TITLE
propan.cli.docs.serve missing from distribution

### ISSUE THREAD

**Issue by bodograumann** [Lancetnik/Propan#68]
**Describe the bug**
In the latest version 0.1.3.0 there is an import for `propan.cli.docs.serve`, but the module is not included in the distribution.
Maybe that is because it is ignored here: https://github.com/Lancetnik/Propan/blob/main/propan/cli/docs/__init__.py#L3 ?

Curiously the `gen.py` exists:
```
$ ls […]/lib/python3.11/site-packages/propan/cli/docs/
app.py  gen.py  __init__.py

$ grep "version" […]/lib/python3.11/site-packages/propan/__about__.py
__version__ = "0.1.3.0"
```

Relevant traceback:
```
    from propan.brokers.rabbit import RabbitBroker, RabbitQueue
/opt/pipx/home/venvs/poetry/lib/python3.11/site-packages/propan/__init__.py:3: in <module>
    from propan.cli.app import *  # noqa: F403
/opt/pipx/home/venvs/poetry/lib/python3.11/site-packages/propan/cli/__init__.py:1: in <module>
    from propan.cli.main import cli
/opt/pipx/home/venvs/poetry/lib/python3.11/site-packages/propan/cli/main.py:10: in <module>
    from propan.cli.docs import docs_app
/opt/pipx/home/venvs/poetry/lib/python3.11/site-packages/propan/cli/docs/__init__.py:1: in <module>
    from propan.cli.docs.app import docs_app
/opt/pipx/home/venvs/poetry/lib/python3.11/site-packages/propan/cli/docs/app.py:12: in <module>
    from propan.cli.docs.serve import serve_docs
E   ModuleNotFoundError: No module named 'propan.cli.docs.serve'
```

**Comment by Lancetnik** [comment]
@bodograumann  I suppose it is connected with `.gitignore` cuz this file reallu doesn't exists at disturbed package

**Comment by Lancetnik** [comment]
And it's fixed with 0.1.3.1
